tests/test_template.py

```
@@ -21,18 +24,35 @@ def test_template_renders(tmp_path: Path, copier: CopierFixture) -> None:
     assert (project / "Cargo.toml").exists()
+    assert (project / "src" / f"{LIB}.rs").exists()
+    project.run("cargo build")
+
+
+def test_template_renders_app_flavour(tmp_path: Path, copier: CopierFixture) -> None:
+    """Template renders app flavour correctly."""
+    project = copier.copy(
+        tmp_path,
+        project_name="AppExample",
+        package_name="app_example",
+        license_year=datetime.now().year,
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)